packages/lab/src/checkbox/CheckboxIcon.css

```diff
@@ -1,80 +1,82 @@
 /* Styles applied to root component */
-.salt-density-high {
-  --checkbox-size: 12px;
-  --checkbox-density-bar-y: 5px;
-  --checkbox-density-bar-height: 3px;
-}
-
-.salt-density-medium {
-  --checkbox-size: 14px;
-  --checkbox-density-bar-y: 6px;
-  --checkbox-density-bar-height: 2px;
-}
 
-.salt-density-low {
-  --checkbox-size: 16px;
-  --checkbox-density-bar-y: 6px;
-  --checkbox-density-bar-height: 2px;
-}
-
-.salt-density-touch {
-  --checkbox-size: 18px;
+.saltCheckboxIcon {
   --checkbox-density-bar-y: 6px;
   --checkbox-density-bar-height: 2px;
-}
 
-.saltCheckboxIcon {
   --checkbox-borderWidth: var(--salt-size-border);
+  --checkbox-size: var(--salt-size-selectable);
   --checkbox-box-offset: calc(var(--checkbox-borderWidth) / 2);
-  --checkbox-height: calc(var(--checkbox-viewbox) - var(--checkbox-borderWidth));
+  /* 14px is used to match the svg's viewbox */
+  --checkbox-box-size: calc(14px - var(--checkbox-borderWidth));
```

Comment:
i think this is fragile, i see the comment but i think this value should be density aware not fixed at 14